packages/renderer/src/lib/ContainerList.svelte

```diff
@@ -377,6 +377,22 @@ function inProgressCallback(container: ContainerInfoUI, inProgress: boolean, sta
   containerGroups = [...containerGroups];
 }
 
+// Go through each container passed in and update the progress
+function composeGroupInProgressCallback(containers: ContainerInfoUI[], inProgress: boolean, state?: string): void {
+  containers.forEach(container => {
+    container.actionInProgress = inProgress;
+    // reset error when starting task
+    if (inProgress) {
+      container.actionError = '';
+    }
```

Comment:
hi, should it be reset only once (so based on the state being `STARTING` ?

here it seems we may reset multiple times ?

Reply:
Yes, it's the same as `inProgressCallback` callback for normal container actions. We reset the container since the callback is targeting a group.

Reply:
but shouldn't we use `if (container.state === 'STARTING)` rather than `if (inProgress)`

Reply:
See: https://github.com/containers/podman-desktop/pull/3390/files#diff-b630c364ef657281e091f3b05ad57e3377b61694bfef381e8dae5834e41661a2R20

We do not check state as inProgress is using the boolean from the "in progress" button presses within ContainerActions / PodActions / ComposeActions.